### PR TITLE
Add bmfont component to list

### PR DESCRIPTION
**Pull request by bryik**
Link to repository: https://github.com/bryik/aframe-bmfont-text-component

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ Third-party components (of the entity-component-system pattern).
 - [Particle System](https://github.com/IdeaSpaceVR/aframe-particle-system-component) - Particle systems (e.g., rain, dust, snow, fire)
 - [WebVR Controller](https://github.com/richardanaya/aframe-webvr-controller) - HTC Vive controllers
 - [Keyboard](https://github.com/richardanaya/aframe-keyboard) - Full VR keyboard for use with WebVR controllers
+- [Bmfont](https://github.com/bryik/aframe-bmfont-text-component) - Renders bitmap/SDF fonts for high quality 2D text
```
Comment (cvan) [inline README.md]: yours seems more complete, but have you seen https://github.com/mattdesl/aframe-bmfont-text/blob/master/index.js? just making sure you saw

Comment (cvan) [inline README.md]: nit 🍆: please hyphenate "high quality 2D text": "high-quality 2D text"

**Comment by bryik** [comment]
@cvan Hyphenation is fixed.

Unfortunately, I didn't see DesLauriers' component until after I'd already written my version. That said, I couldn't get his to work; IIRC certain dependencies are no longer available. 

I hope he resumes working on it as there are properties of three-bmfont-text I didn't expose because they weren't needed for the project I was working on.

**Comment by ngokevin** [comment]
IIRC that component was incomplete/abandoned. Thanks!

**Pull request merged**